config/settings.yml

```diff
@@ -1121,6 +1123,8 @@
         :capacity: 50
         :amqp_port: 5672
         :amqp_heartbeat: 30
+        :ceilometer:
+          :event_types_regex: ".*"
```

Comment:
Please filter here networking events only and remove the networking events from cloud an infra. This should be by default. Otherwise the events will get duplicated and cause duplicate refresh.